lib/livingstyleguide/document.rb

```diff
@@ -12,6 +12,8 @@ class LivingStyleGuide::Document < ::Tilt::Template
   attr_accessor :syntax
   attr_reader :scope
 
+  @@navigation = []
```

Comment:
We tested to import two files and just generates one navigation menu [here](https://github.com/fanxhe/livingstyleguide/blob/add-navigation-menu/lib/livingstyleguide/command_line_interface.rb#L13https://github.com/fanxhe/livingstyleguide/blob/add-navigation-menu/lib/livingstyleguide/command_line_interface.rb%23L13)